### PR TITLE
rpm/hair

### PR DESCRIPTION
**Pull request by r-pm**

### CODE REVIEW

File: src/hair/hair.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./hairStyle.css">
+  <link rel="stylesheet" href="reset.css">
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
```
Comment (juan-rod) [inline src/hair/hair.html:9]: bump these two (reset, font) links before your styles. they need to be loaded first so you can use them.

File: src/hair/hair.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    </nav>
+    <nav id="navlatest">
+      <h3>LATEST</h3>
+    </nav>
+    <nav id="navgiftcards">
+      <h3>GIFT CARDS</h3>
+    </nav>
+    <nav id="navcontact">
+      <h3>CONTACT</h3>
+    </nav>
```
Comment (juan-rod) [inline src/hair/hair.html:42]: add some padding to these nav elements. text touching the edge of the container (and in this case, the screen) is a no-no. 20px is a good number to start with. `padding-left: 20px;`

File: src/hair/hair.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  </header>
+
+  <main>
+    <p>
+      <h2>FIND THE PERFECT STYLIST FOR YOU</h2>
+      <h4>Use our specialized form to generate the best match for your personality & style.</h4>
+    </p>
+    <div>
+      <button class="buttonlocation" id="buttonlocation1">LOCATION 1</button>
+      <button class="buttonlocation" id="buttonlocation2">LOCATION 2</button>
```
Comment (juan-rod) [inline src/hair/hair.html:59]: i don't know what the original image looks like but the button container is trying to swallow the text. lets add some padding!

File: src/hair/hair.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      <ADdress>512.371.7663</ADdress>
+      <address>CONTACT@URBANBETTY.COM</address>
+    </div>
+    <div id="footericons"></div>
+    <div id="footervisit">
+      <h4>VISIT US</h4><br>
+      <address>38TH 1206 W 38TH STREET SUITE 1107 , AUSTIN, TEXAS 78705</address>
+      <address>SOCO 4361 S. CONGRESS AVE UNIT 108, AUSTIN TEXAS 78745</address>
+    </div>
+  </footer>
```
Comment (juan-rod) [inline src/hair/hair.html:76]: when in mobile view, stack these child elements of footer on top of each other. in the media query change the flex-direction to column. `flex-direction: column;`

File: src/hair/hairStyle.css [hunk 5]
```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:400,700&display=swap');
+
+body {
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./img/mainbackgroundimg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  /* how do i know when to do vh vs %? */
```
Comment (juan-rod) [inline src/hair/hairStyle.css:9]: you don't. you do what works and what looks good. lol. height is a finicky property. there may be an actual rule about when to use either but I have researched. if you do, please let me and the class know!

File: src/hair/hairStyle.css [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+aside {
+  grid-area: 1/1/-1/2;
+  display: grid;
+  grid-template-columns: 200px;
+  grid-template-rows: inherit;
+  /* background-color: rgb(73, 72, 71); 
+  here i learned cant have bg-color & bkg rgba regardless of order*/
+  background:rgba(251,249,244,0.9);
```
Comment (juan-rod) [inline src/hair/hairStyle.css:23]: oh yeah, background is short for all the other background properties ( color, size, position...) you can use it when you want to put them all on one line. [background property](https://www.w3schools.com/cssref/css3_pr_background.asp)

File: src/hair/hairStyle.css [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+    grid-column: 1/-1;
+    grid-row: 1/2;
+    background-image: url(./img/bettylogo.png);
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    /* need to set max heights/widths for img in diff screens */
+  }
+  /* um could i have done this easier? tried to grid w.o assigning ea id but they shifted */
+  .navabout {
```
Comment (juan-rod) [inline src/hair/hairStyle.css:37]: i would have grouped these all into one nav div. and flexbox the thing.

**Review by juan-rod — APPROVED** [review]
great work!!! we love progress! merge when ready!

File: src/hair/hair.html (hunk 1 above)
Comment (r-pm) [reply src/hair/hair.html:9]: I'm finally reading all the comments so I will not make you write this comment again 😅

**Pull request merged**